MultiSnap_timelapse_ef_230209_3.bsh

```diff
@@ -66,15 +67,15 @@ Color[] colors = {Color.RED};
 // File locations (high reso) _220523gy�@
 dirhigh = "G:/MSM temp" + "/" + Atime.substring(24,25) + Atime.substring(25,26) + Atime.substring(26,27) + Atime.substring(27,28)
 	    + Atime.substring(4,5) + Atime.substring(5,6) + Atime.substring(6,7) + Atime.substring(8,9) + Atime.substring(9,10)
-		 + "_" + Atime.substring(11,12) + Atime.substring(12,13) + Atime.substring(14,15) + Atime.substring(15,16) + "high"; //�f�B���N�g���̏ꏊ�w��c���{��
+		 + "_" + Atime.substring(11,12) + Atime.substring(12,13) + Atime.substring(14,15) + Atime.substring(15,16) + Atime.substring(17,18) + Atime.substring(18,19) + "high"; //�f�B���N�g���̏ꏊ�w��c���{��
 StoreHigh = mm.data().createMultipageTIFFDatastore(dirhigh, false, false); // Create datastore
 targetStore = mm.data().createRAMDatastore(); // Use RAM datastore without saving
 DisplayHigh = mm.displays().createDisplay(StoreHigh); // Create displayer for datastore
 
 // File locations (wide field) _220523gy
 dirwide = "G:/MSM temp" + "/" + Atime.substring(24,25) + Atime.substring(25,26) + Atime.substring(26,27) + Atime.substring(27,28)
 	    + Atime.substring(4,5) + Atime.substring(5,6) + Atime.substring(6,7) + Atime.substring(8,9) + Atime.substring(9,10)
-		 + "_" + Atime.substring(11,12) + Atime.substring(12,13) + Atime.substring(14,15) + Atime.substring(15,16) + "wide";//�f�B���N�g���̏ꏊ�w��c�L����
+		 + "_" + Atime.substring(11,12) + Atime.substring(12,13) + Atime.substring(14,15) + Atime.substring(15,16) + Atime.substring(17,18) + Atime.substring(18,19) + "wide";//�f�B���N�g���̏ꏊ�w��c�L����
```

Comment:
these codes are split to avoid ":" in file name. If the time tag does not contain ":", these codes can be fused as 
`dirwide = "G:/MSM temp" + "/" + Atime.substring(24,28) + Atime.substring(4,7) + Atime.substring(8,10) + "_" + Atime.substring(11,13) + Atime.substring(14,16) + Atime.substring(17,19) + "wide";`

Reply:
I agree with you. I will change as your suggestion.